docs/src/tutorials/raise-alarm.md

```diff
@@ -100,6 +139,50 @@ For example the `temperature_high` alarm with `critical` severity described in t
 
 If the alarm is raised from a child device, the payload is published to `c8y/s/us/<child-device-id>` topic instead.
 
+If an alarm contains a custom fragment then the alarm message will be converted to `cumulocity json`
+format and then will be published on to `c8y/alarm/alarms/create` topic.
```

Comment:
```suggestion
If an alarm contains a custom fragment then the alarm message will be converted to `cumulocity json` format and then it will be published to the `c8y/alarm/alarms/create` topic.
```